tests/time_tests/test_runner/conftest.py

```diff
@@ -186,7 +202,8 @@ def validate_test_case(request, test_info):
             "precision": {"type": "string"},
             "framework": {"type": "string"}
         })
-    test_info["submit_to_db"] = True
+        schema["properties"]["model"]["required"].extend(["name", "precision", "framework"])
```

Comment:
the schema should be constant. I'd even prefer it to be a string we will parse on startup. Shoudl we utilize "dependencies" https://json-schema.org/understanding-json-schema/reference/object.html#dependencies, https://stackoverflow.com/questions/38717933/jsonschema-attribute-conditionally-required

Reply:
Done